### PR TITLE
refactor: move transaction history toggle to settings page

### PR DESCRIPTION
**Pull request by dated**
## Summary

https://app.clickup.com/t/e0x7j6

> ready to be merged after https://github.com/ArkEcosystem/desktop-wallet/pull/3441

## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Comment by faustbrian** [comment]
Task linked: [CU-e0x7j6 [Settings Page] Move 'Transaction History' option to setting page](https://app.clickup.com/t/e0x7j6)

**Comment by samharperpittam** [comment]
@dated - My Transaction History table is enabled by default, but looking in Settings it's shown as disabled. :hmhm:

If you create a new profile you'll see the same thing, the table should show as enabled within settings by default.

**Comment by dated** [comment]
Thanks @samharperpittam, fixed in 0b1533c

**Review by samharperpittam — APPROVED** [review]

File: src/domains/setting/i18n.ts [hunk 1]
```diff
@@ -51,6 +51,14 @@ export const translations: { [key: string]: any } = {
 				TITLE: "Error Reporting",
 				DESCRIPTION: "Automatically send reports to help to improve our products.",
 			},
+			TRANSACTION_HISTORY: {
+				TITLE: "Portfolio Transaction History",
+				DESCRIPTION: "Enables transaction history on Portfolio page",
+			},
+			UPDATE_LEDGER: {
+				TITLE: "Update Ledger in Background",
+				DESCRIPTION: "Allows Ledger wallets to automatically update and sync data in the background.",
```
Comment (ItsANameToo) [inline src/domains/setting/i18n.ts:60]: why did this return 🤔

Comment (dated) [reply src/domains/setting/i18n.ts:60]: :bleh:

Comment (dated) [reply src/domains/setting/i18n.ts]: i think i moved those keys around before, so they didn't get purged with the merges. they are removed now.

**Pull request merged**